[2025-03-13T21:24:20.628+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-13T21:24:20.635+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: open_weather_etl.extract manual__2025-03-13T21:24:19.232990+00:00 [queued]>
[2025-03-13T21:24:20.639+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: open_weather_etl.extract manual__2025-03-13T21:24:19.232990+00:00 [queued]>
[2025-03-13T21:24:20.640+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 2
[2025-03-13T21:24:20.819+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): extract> on 2025-03-13 21:24:19.232990+00:00
[2025-03-13T21:24:20.824+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=176) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-03-13T21:24:20.825+0000] {standard_task_runner.py:72} INFO - Started process 177 to run task
[2025-03-13T21:24:20.826+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'open_weather_etl', 'extract', 'manual__2025-03-13T21:24:19.232990+00:00', '--job-id', '31', '--raw', '--subdir', 'DAGS_FOLDER/test_dag.py', '--cfg-path', '/tmp/tmp4m98xfac']
[2025-03-13T21:24:20.827+0000] {standard_task_runner.py:105} INFO - Job 31: Subtask extract
[2025-03-13T21:24:20.861+0000] {task_command.py:467} INFO - Running <TaskInstance: open_weather_etl.extract manual__2025-03-13T21:24:19.232990+00:00 [running]> on host 7499a3ac39c6
[2025-03-13T21:24:20.891+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='your_email@example.com' AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='open_weather_etl' AIRFLOW_CTX_TASK_ID='extract' AIRFLOW_CTX_EXECUTION_DATE='2025-03-13T21:24:19.232990+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-03-13T21:24:19.232990+00:00'
[2025-03-13T21:24:20.892+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-03-13T21:24:20.901+0000] {logging_mixin.py:190} INFO - Starting extraction process from Open Weather API...
[2025-03-13T21:24:20.912+0000] {logging_mixin.py:190} INFO - Testing the funciton inside the container
[2025-03-13T21:24:20.912+0000] {logging_mixin.py:190} INFO - Extraction complete.
[2025-03-13T21:24:20.918+0000] {python.py:240} INFO - Done. Returned value was: Extraction completed successfully!
[2025-03-13T21:24:20.924+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-03-13T21:24:20.924+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=open_weather_etl, task_id=extract, run_id=manual__2025-03-13T21:24:19.232990+00:00, execution_date=20250313T212419, start_date=20250313T212420, end_date=20250313T212420
[2025-03-13T21:24:20.966+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-13T21:24:20.980+0000] {taskinstance.py:3901} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-03-13T21:24:20.981+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
